### PR TITLE
Hw03 mongodb

### PR DESCRIPTION
**Pull request by szkic**

### CODE REVIEW

File: controller/index.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    if (editContact) {
+      res.status(200).json({
+        message: "contact edited",
+        data: { editContact },
+      });
+    } else {
+      res.status(404).json({ message: "Not found" });
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (AdamChodowiecGoIt91) [inline controller/index.js]: what response will be returned if your `catch` will be executed?

File: controller/index.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    if (editFavorite) {
+      res.status(200).json({
+        message: "contact edited",
+        data: editFavorite,
+      });
+    } else {
+      res.status(404).json({ message: "Not found" });
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (AdamChodowiecGoIt91) [inline controller/index.js]: what response will be returned if your `catch` will be executed?

File: controller/index.js (hunk 1 above)
Comment (szkic) [reply controller/index.js]: added response status and message

File: controller/index.js (hunk 2 above)
Comment (szkic) [reply controller/index.js]: added response status and message

**Pull request merged**